doc/extension_store/manifest.jsonc

```diff
@@ -0,0 +1,153 @@
+            "regex": "jdbc:\\w+://...",
+            "description": "JDBC URL of the PostgreSQL database",
+            "scope": "connection",
+            "default": "jdbc:postgresql://<host>:<port>/<database name>?ssl=true&sslfactory=org.postgresql.ssl.DefaultJavaSSLFactory"
+        },
+        "user": {
+            "type": "string",
+            "description": "Name of the PostgreSQL user",
+            "scope": "connection"
+        },
+        "password": {
+            "type": "password", // Password input field
+            "description": "Password of the PostgreSQL user",
+            "scope": "connection"
+        },
+        "virtualSchemaName": {
+            "type": "string",
+            "description": "Name of the virtual schema",
+            "default": "VIRTUAL_POSTGRES_SCHEMA",
+            "readOnlyAfterInstallation": true,
+            "scope": "connection"
+        },
+        "postgresCatalogName": {
+            "type": "string",
+            "description": "Name of the PostgreSQL catalog / database",
+            "scope": "connection",
+            "default": ""
+        },
+        "postgresSchemaName": {
+            "type": "string",
+            "description": "Name of the PostgreSQL schema",
+            "scope": "connection",
+            "default": ""
+        },
+        "identifierMapping": {
+            "type": "select", // Show select list with default value
+            "description": "Postgres identifier mapping",
+            "scope": "connection",
+            "values": [
+                {
+                    "name": "CONVERT_TO_UPPER",
+                    "description": "Automatic identifier handling (do not use quoted identifiers!)",
+                    "default": true
+                },
+                {
+                    "name": "PRESERVE_ORIGINAL_CASE",
+                    "description": "Don't change case"
+                }
+            ]
+        },
+        "ignoreErrors": {
+            "type": "select",
+            "description": "Ignore postgres errors",
+            "scope": "connection",
+            "values": [
+                {
+                    "name": "POSTGRESQL_UPPERCASE_TABLES",
+                    "description": "Ignore upper case table names in Postgres",
+                    "default": false
+                },
+                {
+                    "name": "",
+                    "description": "Don't ignore upper case table names in Postgres",
+                    "default": true
+                }
+            ]
+        }
+    },
+    "sqlScripts": {
+        "create": {
+            "configuration": [
+                "CREATE IF NOT EXISTS SCHEMA \"${param.adapterSchema}\"",
+                "CREATE OR REPLACE JAVA ADAPTER SCRIPT \"${param.adapterSchema}\".\"${param.adapterScriptName}\" AS %scriptclass com.exasol.adapter.RequestDispatcher; %jar ${bucketFsUpload.jdbcDriver.bucketFsPath}; %jar ${bucketFsUpload.virtualSchemaAdapter.bucketFsPath};",
```

Comment:
oh, I found a case where simple SQL is not enough. Our python extensions, usually, bring a whole language container. For that, we need to add a new language via alter session. This means, we first need to fetch the current script_languages string via `SELECT * FROM SYS.EXA_PARAMETERS WHERE ...` and then append our new container. Maybe, this has to be it own mechanism.
https://docs.exasol.com/db/latest/database_concepts/udf_scripts/adding_new_packages_script_languages.htm

Reply:
Add to open points